pkg/controller/volume/attachdetach/reconciler/reconciler_test.go

```diff
@@ -591,9 +676,11 @@ func Test_ReportMultiAttachError(t *testing.T) {
 			fakeRecorder,
 			false, /* checkNodeCapabilitiesBeforeMount */
 			fakeHandler))
+		informerFactory := informers.NewSharedInformerFactory(fakeKubeClient, controller.NoResyncPeriodFunc())
+		nodeLister := informerFactory.Core().V1().Nodes().Lister()
 		nsu := statusupdater.NewFakeNodeStatusUpdater(false /* returnError */)
 		rc := NewReconciler(
-			reconcilerLoopPeriod, maxWaitForUnmountDuration, syncLoopPeriod, false, dsw, asw, ad, nsu, fakeRecorder)
+			reconcilerLoopPeriod, maxWaitForUnmountDuration, syncLoopPeriod, false, dsw, asw, ad, nsu, nodeLister, fakeRecorder)
```

Comment:
Please add "Disruptive" E2E tests to prevent future regressions.

Reply:
Sure.

Reply:
there are already some disruptive tests. Feel free to expand them. Heads up - currently our e2e infra. does not support running disruptive tests on AWS, so your only realistic choice is to test against gce.